### PR TITLE
feat(i18n): add Arabic language config to all langs

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

ref #48101

### CODE REVIEW

File: config/i18n/all-langs.ts [hunk 1]
```diff
@@ -139,6 +141,11 @@ export const auditedCerts = {
     SuperBlocks.RespWebDesign,
     SuperBlocks.JsAlgoDataStruct,
     SuperBlocks.FrontEndDevLibs
+  ],
+  arabic: [
+    SuperBlocks.RespWebDesign,
```
Comment (Sboonny) [inline config/i18n/all-langs.ts:146]:
```suggestion
    SuperBlocks.RespWebDesignNew,

**Comment by Sboonny** [comment]
the new RWD is fully released, so I don't need to change `languagesWithAuditedBetaReleases`? 🤔

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Yes, if the new RWD is translated, you'll need to add it to the `languagesWithAuditedBetaReleases` to move the new RWD to the top of the map.

This is also blocked as we have not enabled the curriculum downloads for Arabic and have 0 translations in this repository.

**Review by naomi-lgbt — APPROVED** [review]
With the hidden lang option, this should be good to merge.

We'll need to coordinate with @raisedadead for the deployment of the new instance.

**Review by sidemt — APPROVED** [review]
LGTM
For future reference: algolia-locale-setup.ts is updated in PR #48129

**Pull request merged**